ld.hugetlbfs

```diff
@@ -63,12 +65,19 @@ while [ -n "$1" ]; do
 	    args=("${args[@]}" "$@")
 	    break
 	    ;;
+	@*)
+		# remove the trailing @
+		FILE=${arg:1}
+		while read -r file_arg; do
+			arglist+=("$file_arg")
```

Comment:
I am not sure this handles the quoting correctly. And I suspect you might end up with a huge command line in some cases which then will fail the exec.

Reply:
Ahh, now I know why `@file` is necessary. I just made another commit to extract related arguments from file, and keep other arguments still in the file. Could you please check if it is alright?